### PR TITLE
droot 0.2.0

### PR DESCRIPTION
**Pull request by chansuke**

### CODE REVIEW

File: Library/Formula/droot.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require "language/go"
+
+class Droot < Formula
+  homepage "https://github.com/yuuki1/droot"
+  url "https://github.com/yuuki1/droot.git", :tag => "v0.2.0", :revision => "8e7dd1440f06ab1a1cc7205ab0d04bf406c15677"
```
Comment (DomT4) [inline Library/Formula/droot.rb]: `:tag` and `:revision` should live on separate lines for readability.

File: Library/Formula/droot.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  head "https://github.com/yuuki1/droot.git", :branch => "master"
```
Comment (DomT4) [inline Library/Formula/droot.rb]: Don't need to specify `master` as the branch; we assume that to be the default.

File: Library/Formula/droot.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/yuuki1/"
+    ln_sf buildpath, buildpath/"src/github.com/yuuki1/droot/"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", "droot"
+    bin.install "droot"
+
```
Comment (DomT4) [inline Library/Formula/droot.rb]: Don't need this blank line.

File: Library/Formula/droot.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  head "https://github.com/yuuki1/droot.git", :branch => "master"
+
+  depends_on "go" => :build
+  depends_on :hg => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/yuuki1/"
+    ln_sf buildpath, buildpath/"src/github.com/yuuki1/droot/"
+    Language::Go.stage_deps resources, buildpath/"src"
```
Comment (DomT4) [inline Library/Formula/droot.rb]: You haven't actually included any resources in the formula? Do you need to, or?

**Comment by DomT4** [comment]
Bunch of audit stuff:

```
==> Checking out tag v0.2.0
Error: v0.2.0 tag should be 8e7dd1440f06ab1a1cc7205ab0d04bf406c15677
but is actually 0fa1954b3368e69d06f01b82cbb311d72683aaff

==> audit problems
droot:
 * Formula should have a desc (Description).
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
 * GitHub repository too new (<30 days old)
```

**Comment by mikemcquaid** [comment]
> * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
>  * GitHub repository too new (<30 days old)

Passing on this for now because of these.

**Pull request closed without merging**